src/utils/passportScanning.test.ts

```diff
@@ -88,14 +88,24 @@ describe("passportScanning", () => {
       });
 
       describe("cases without passportId", () => {
-        it("should return empty string if it does not contain passportId", () => {
+        it("should return empty string if data is null", () => {
```

Comment:
to match the `case with passportId`, should we also add test for data is just a string that's not `passportId` object?

```
event = {
  data: JSON.stringify("invalid-string")
};
```

Reply:
should have the same result with `passportId === undefined`

Reply:
Have added this test case!